doc/languages-frameworks/javascript.section.md

```diff
@@ -342,8 +342,8 @@ stdenv.mkDerivation (finalAttrs: {
 })
 ```
 
-NOTE: It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version.
-
+It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version. The fetchDeps hook will abort if the used pnpm version is too old.
+If desired, that version requirement can be relaxed via `removePackageManagerVersion`.
```

Comment:
The phrasing is still honestly a bit odd. Also, it might be better to rename the parameter to something like `relaxPnpmVersion` 

```suggestion
It is highly recommended to use a pinned version of pnpm (i.e. `pnpm_8` or `pnpm_9`), to increase future reproducibility. It might also be required to use an older version, if the package needs support for a certain lock file version.
Unless the `relaxPnpmVersion` parameter is set to true, the fetchDeps hook will abort if the used pnpm version is older than the `lockfileVersion` specified in the lockfile.
```